R/pkg/R/DataFrame.R

```diff
@@ -890,7 +890,7 @@ setMethod("toJSON",
 #'             save mode (it is 'error' by default)
 #' @param ... additional argument(s) passed to the method.
 #'            You can find the JSON-specific options for writing JSON files in
-#'            \url{
+#'            \href{
```

Comment:
Oh @zero323. actually I intentionally did not replace these `url`s as a workaround for now. CRAN admin complaint about `href`s when we submitted SparkR package before (see https://github.com/apache/spark/pull/31058). The problem is that I don't know what tool they use. IIRC CRAN check did not automatically check this either.

Reply:
I mean, I fixed `url` in that PR too but my guess was that CRAN only checked `href`. So we should probably make sure these `href` does not have the same problem described in https://github.com/apache/spark/pull/31058

Reply:
Oh well... Let me close it :)

Reply:
yeah .. this is a bit frustrating TBH, and we're considering about stopping the releases on CRAN for now ..

Reply:
> yeah .. this is a bit frustrating TBH, and we're considering about stopping the releases on CRAN for now ..

I actually have some thoughts about it, but that's for another day.

Reply:
Also, sorry for the noise....

Reply:
no worries!